dll/win32/imm32/imm.c

```diff
@@ -36,14 +34,13 @@
 #include "winnls.h"
 #include "winreg.h"
```

Comment:
Then I suggest you fix as well the way Wine used to do their includes, and use the angle brackets since these headers are the standard ones.

Reply:
Used angles.